### ISSUE TITLE
Iframe focus support

### ISSUE THREAD

**Issue by WilcoFiers** [auto-wcag/auto-wcag#2292]
## Accessibility support

There are several known accessibility support issues with iframe accessible names:

- Some browsers instantly redirect focus from iframe elements to the first focusable element inside that iframe. This redirect makes it appear as though the iframe never receives focus, even if the iframe has a non-negative `tabindex` attribute value.
- Not all browsers redirect focus on iframe elements. This ensures that the contents of iframe elements can be scrolled and accessed by using the keyboard. This must not be circumvented by using a negative tabindex, as this will make the iframe completely inaccessible for keyboard navigation.

## ACT Rules

- [Iframe element has non-empty accessible name - cae760](https://www.w3.org/WAI/standards-guidelines/act/rules/cae760/proposed/#accessibility-support)

### Test case 1

Testing iframe accessible name support across different naming methods

```html
TODO
```

#### Test instructions

TODO

## Test results

Please report test results in a comment below. Please use the following format:

```md
## Test case 1

- Date: ...
- Operating system: ..., version ...
- Browser: ..., version ...
- Screen reader: ..., version ...
- Focus behavior: ... (redirected/not redirected)
- Test result: Passed / Failed (if failed, how was it different from expected?)
```
